website/docs/sql/datatype/array.md

```diff
@@ -0,0 +1,84 @@
+# Array Type
+
+An array is a sequential collection of elements packed into a single SQL value.
+Arrays are useful to model dependent information with an inherent ordering in a single row – for example, time-series data from longitudinal studies, or embedding vectors from machine learning models. 
+
+Generally, array values are denoted in curly braces with elements seperated by commas, like so:
+
+```
+{'Lorem', 'ipsum', 'dolor', 'sit', 'amed'}
```

Comment:
Using double quotes for the string values inside the array allows users to copy paste this into a single quote escaped array literal.

```suggestion
{"Lorem", "ipsum", "dolor", "sit", "amed"}
```

Reply:
[9fabdcd](https://github.com/tableau/hyper-db/pull/110/commits/9fabdcd8f563d54f052cc80085c08c87b54f630a)